### PR TITLE
put the AKS content 'At the top level between "Manage FMW Infrastructure domains" and "CI/CD Considerations"'

### PR DESCRIPTION
**Pull request by edburns**
On branch edburns-msft-dd-1426715-wls-aks-docs Per @mriccell and @rosemarymarano @rosemarymarano, put the AKS content 'At the top level between "Manage FMW Infrastructure domains" and "CI/CD Considerations"'

modified:   documentation/3.3/content/samples/azure-kubernetes-service/_index.md
new file:   documentation/3.3/content/userguide/aks/_index.md
modified:   documentation/3.3/content/userguide/base-images/_index.md
modified:   documentation/3.3/content/userguide/cicd/_index.md
modified:   documentation/3.3/content/userguide/istio/istio.md
modified:   documentation/3.3/content/userguide/kubernetes/k8s-setup.md
modified:   documentation/3.3/content/userguide/managing-domains/_index.md
modified:   documentation/3.3/content/userguide/managing-fmw-domains/_index.md
modified:   documentation/3.3/content/userguide/managing-operators/_index.md
modified:   documentation/3.3/content/userguide/platforms/environments.md
modified:   documentation/staging/content/samples/azure-kubernetes-service/_index.md

- Re-order weights to preserve layout.  Note, this commit also
  corrects an apparent conflict that was relying on lexical ordering.

   - These two had `weight: 3`:

      ```
      ./prepare/_index.md
      ./platforms/environments.md
      ```

   - But "Operator prerequisites" sorted before "Setup checklist".

   - This commit resolves this conflict by incrementing
     `environments`, and the subsequent chapters as well.

      ```
      +++ b/documentation/3.3/content/userguide/platforms/environments.md
      @@ -2,7 +2,7 @@
       title: "Supported platforms"
       date: 2019-02-23T16:40:54-05:00
       description: "See the operator supported environments."
      -weight: 3
      +weight: 4
      ```

Signed-off-by: Ed Burns <edburns@microsoft.com>

### CODE REVIEW

File: documentation/3.3/content/userguide/aks/_index.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+   - [DNS Configuration](#dns-configuration)
+   - [Database](#database)
+   - [Review + create](#review--create)
+   
+
+#### Introduction
+
+{{< readfile file="/samples/azure-kubernetes-service/includes/aks-value-prop.txt" >}}
+
+This section documents the Azure Marketplace offer that makes it easy to get started with WebLogic Server on Azure. The offer handles all the initial setup, creating the AKS cluster, container registry, WebLogic Kubernetes Operator installation, and domain creation using the model-in-image domain home source type. For complete details on domain home source types, see [Choose a domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}).
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/3.3/content/userguide/aks/_index.md:24]: So, this comment was not incorporated. Was that on purpose?

File: documentation/3.3/content/userguide/aks/_index.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+| Field | Description |
+|-------|-------------|
+| Region | Select an Azure region from the drop-down list. |
+
+##### Credentials for WebLogic
+
+| Field | Description |
+|-------|-------------|
+| Username for WebLogic Administrator | Enter a user name to access the WebLogic Administration Console which is started automatically after the provisioning. For more information about the WebLogic Administration Console, see [Overview of Administration Consoles](https://docs.oracle.com/pls/topic/lookup?ctx=en/middleware/standalone/weblogic-server/wlazu&id=INTRO-GUID-CC01963A-6073-4ABD-BC5F-5C509CA1EA90) in _Understanding Oracle WebLogic Server_. |
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/3.3/content/userguide/aks/_index.md:50]: Not incorporated.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+| Password for WebLogic Administrator | Enter a password to access the WebLogic Administration Console. |
+| Confirm password | Re-enter the value of the preceding field. |
+| Password for WebLogic Deploy Tooling runtime encrytion | The deployment uses Weblogic Deploy Tooling, including the capability to encrypt the model. This password is used for that encrption. For more information, see [Encrypt Model Tool](https://oracle.github.io/weblogic-deploy-tooling/userguide/tools/encrypt/). |
+| Confirm password | Re-enter the value of the preceding field. |
+| User assigned managed identity | The deployment requires a user-assigned managed identity with the **Contributor** or **Owner** role in the subscription referenced above.  For more information please see [Create, list, delete, or assign a role to a user-assigned managed identity using the Azure portal](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-manage-ua-identity-portal). |
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/3.3/content/userguide/aks/_index.md:55]: Missed some of these.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+##### Optional Basic Configuration
+
+| Field | Description |
+|-------|-------------|
+| Accept defaults for optional configuration? | If you want to retain the default values for the optional configuration, such as **Name prefix for Managed Server**, **WebLogic Domain Name** and others, set the toggle button to **Yes**, and click **Next: Configure AKS cluster**. If you want to specify different values for the optional configuration, set the toggle button to **No**, and enter the following details. |
+| Name prefix for Managed Server | Enter a prefix for the Managed Server name. |
+| WebLogic Domain Name | Enter the name of the domain that will be created by the offer. |
+| Cluster size |  |
+|Custom Java Options to start WebLogic Server | Java VM arguments passed to the invocation of WebLogic Server. For more information, see the [FAQ]({{< relref "/faq/resource-settings/_index.md" >}}). |
+|Enable T3 tunneling for Admin Server| If checked, configure the necessary settings to enable T3 tunneling to the Admin Server.  For more details see [External network access security]({{< relref "/security/domain-security/weblogic-channels.md" >}}).|
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/3.3/content/userguide/aks/_index.md:66]: Not incorporated.

**Review by rosemarymarano — COMMENTED** [review]
I'll stop here. AFAICT, many edits were not incorporated. If you prefer, I can take the doc (after it's merged) and fix them all.

File: documentation/3.3/content/userguide/aks/_index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:55]: Fixed in 7477e9be62 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:66]: Fixed in c98ba6ac55 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+|Enable T3 tunneling for WebLogic cluster| If checked, configure the necessary settings to enable T3 tunneling to the WebLogic Server cluster.  For more details see [External network access security]({{< relref "/security/domain-security/weblogic-channels.md" >}}).|
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:67]: Fixed in c98ba6ac55f707bb5de539e33e16cac425f8faca . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+
+When you are satisfied with your selections select **Next : Configure AKS cluster**.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:69]: Fixed in b26c5f98d1 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+Use the **Configure AKS Cluster** blade to configure fundamental details of how Oracle WebLogic Server runs on AKS. To do this, enter the values for the fields listed in the following tables.
+
+##### Azure Kubernetes Service
+
+This section allows you to configure some options about the AKS which will run WebLogic Server.
+
+| Field | Description |
+|-------|-------------|
+|Create a new AKS cluster| If set to **Yes**, the deployment will create a new AKS cluster resource in the specified resource group. If set to **No**, you have the opportunity to select an existing AKS cluster, into which the deployment is configured. Note: the offer assumes the existing AKS cluster has no WebLogic related deployments. |
+| Node count | The initial number of nodes in the AKS cluster. This value can be changed after deployment. For information, see [Scaling]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling.md" >}}) |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:82]: Fixed in a82ff8f498 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+| Node size | The default VM size is 2x Standard DSv2, 2 vcpus, 7 GB memory. If you want to select a different VM size, click Change Size, select the size from the list (for example, A3) on the Select a VM size page, and click Select. For more information about sizing the virtual machine, see [Azure documentation on Sizes](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs).|
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md]: Fixed in 448ec221b9 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+|Enable Container insights| If checked, configure the necessary settings to integrate with Container insights. For more information see [Container insights overview](https://aka.ms/wls-aks-container-insights).|
+|Create Persistent Volume using Azure File share service|If checked, configure the necessary settings to mount a persistent volume to the nodes of the AKS cluster. For more information see [Persistent storage]({{< relref "/userguide/managing-domains/persistent-storage/_index.md" >}}).|
+
+##### Image selection
+
+This section allows you to configure the image that is deployed using the model-in-image domain home source type. There are several options for the WebLogic image and the application image deployed therein.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:89]: Thank you [Ms. Strunk](https://smile.amazon.com/s?k=adios+strunk+and+white+6th+edition&sprefix=adios+strunk%2Caps%2C97&ref=nb_sb_ss_ts-doa-p_1_12).   Fixed in db7f9ccb77 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+|Create a new AKS cluster| If set to **Yes**, the deployment will create a new AKS cluster resource in the specified resource group. If set to **No**, you have the opportunity to select an existing AKS cluster, into which the deployment is configured. Note: the offer assumes the existing AKS cluster has no WebLogic related deployments. |
+| Node count | The initial number of nodes in the AKS cluster. This value can be changed after deployment. For information, see [Scaling]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling.md" >}}) |
+| Node size | The default VM size is 2x Standard DSv2, 2 vcpus, 7 GB memory. If you want to select a different VM size, click Change Size, select the size from the list (for example, A3) on the Select a VM size page, and click Select. For more information about sizing the virtual machine, see [Azure documentation on Sizes](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs).|
+|Enable Container insights| If checked, configure the necessary settings to integrate with Container insights. For more information see [Container insights overview](https://aka.ms/wls-aks-container-insights).|
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:84]: Fixed in c98ba6ac55f707bb . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:24]: Fixed in 157d8f9724 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 11]
```diff
@@ -0,0 +1,278 @@
...
+
+##### Image selection
+
+This section allows you to configure the image that is deployed using the model-in-image domain home source type. There are several options for the WebLogic image and the application image deployed therein.
+
+| Field | Description |
+|-------|-------------|
+| Use a pre-existing WebLogic Server Docker image from Oracle Container Registry? | If set to **Yes**, the subsequent options are constrained to only allow selecting from a set of pre-existing WebLogic Server Docker images stored in the Oracle Container Registry. If set to **No**, the user may refer to a pre-existing Azure Container Registry, and must specify the Docker tag of the WebLogic Server image within that registry that will be used to create the domain. The specified image is assumed to be compatible with the WebLogic Kubernetes Operator. For more about WebLogic Server images see [WebLogic Server images]({{< relref "/userguide/base-images/_index.md" >}}).|
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md]: Fixed in 46f4fd0190 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 12]
```diff
@@ -0,0 +1,278 @@
...
+|Create a new Azure Container Registry to store application images?|If set to **Yes**, the offer will create a new Azure Container Registry (ACR) to hold the Docker images for use in the deployment.  If set to **No**, you must specify an existing ACR. In this case, you must be sure the selected ACR has the admin account enabled. For details, please see [Admin account](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-authentication?WT.mc_id=Portal-Microsoft_Azure_CreateUIDef&tabs=azure-cli#admin-account). |
+| Select existing ACR instance | This option is only shown if **Use a pre-existing WebLogic Server Docker image from Oracle Container Registry?** is set to **No**. If visible, select an existing Acure Container Registry instance. |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:95]: Fixed in ea62f86192 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 13]
```diff
@@ -0,0 +1,278 @@
...
+| Please provide the image path | This option is only shown if **Use a pre-existing WebLogic Server Docker image from Oracle Container Registry?** is set to **No**. If visible, the value must be a fully qualified Docker tag of an image within the specified ACR. |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:96]: Fixed in ea62f86192 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 14]
```diff
@@ -0,0 +1,278 @@
...
+| Username for Oracle Single Sign-On authentication | The Oracle Single Sign-on account user name for which the Terms and Restrictions for the selected WebLogic Server image have been accepted. |
+| Password for Oracle Single Sign-On authentication | The password for that account. | 
+| Confirm password | Re-enter the value of the preceding field. |
+| Select WebLogic Server Docker tag | Select one of the supported images. |
+
+##### Java EE Application
+
+Options in this section enable you to deploy a Java EE Application along with the WebLogic Server deployment.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:104]: Fixed in db7f9ccb779 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 15]
```diff
@@ -0,0 +1,278 @@
...
+
+| Field | Description |
+|-------|-------------|
+| Deploy your application package? | If set to **Yes**, you must specify a Java EE WAR, EAR, or JAR file suitable for deployment with the selected version of WebLogic Server. If set to **No**, no application is deployed.| 
+| Application package (.war,.ear,.jar) | The **Browse** button enables you to browse and select a file from a pre-existing Azure Storage Account and Storage Container within that account.  To learn how to create a Storage Account and Container, see [Create a storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal). |
+| Fail deployment if application does not become ACTIVE. | If checked, the deployment will wait for the deployed application to reach the **ACTIVE** state and fail the deployment if it does not. For more details, see the [Oracle documentation](https://aka.ms/wls-aks-deployment-state) |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:110]: Fixed in 362b24c75d87 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 16]
```diff
@@ -0,0 +1,278 @@
...
+| Number of application replicas | The initial value of the `replicas` field of the Domain. For information, see [Scaling]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling.md" >}}). |
+
+When you are satisfied with your selections select **Next : TLS/SSL configuration**.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:113]: Fixed in b26c5f98d1c . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 17]
```diff
@@ -0,0 +1,278 @@
...
+
+#### TLS/SSL configuration
+
+The **TLS/SSL configuration** blade enables you to configure Oracle WebLogic Server Administration Console on a secure HTTPS port, with your own SSL certificate provided by a Certifying Authority (CA).
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:117]: Fixed in db7f9ccb779 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 18]
```diff
@@ -0,0 +1,278 @@
...
+
+Select **Yes** or **No** for the option **onfigure WebLogic Administration Console, Remote Console, cluster and custom T3 channel to use HTTPS (Secure) ports, with your own TLS/SSL certificate.** based on your preference. If you select No, you don't have to provide any details, and can proceed by selecting **Next : Networking** >. If you select **Yes**, you can choose to provide the required configuration details by either uploading existing keystores or by using keystores stored in Azure Key Vault.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:119]: Fixed in e15fc2aec61 and 19e6749607 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 19]
```diff
@@ -0,0 +1,278 @@
...
+| Trust KeyStore Data file(.jks,.p12) | Upload a custom trust keystore data file by doing the following: {{< line_break >}} 1. Click on the file icon. {{< line_break >}} 2. Navigate to the folder where the identity keystore file resides, and select the file. {{< line_break >}} 3. Click Open. |
+| Password | Enter the password for the custom trust keystore. |
+| Confirm password | Re-enter the value of the preceding field. |
+| The Identity KeyStore type (JKS,PKCS12) | Select the type of custom trust keystore. The supported values are JKS and PKCS12. |
+
+When you are satisfied with your selections select **Next : Networking**.
+
+#### Networking
+
+This blade allows configuring options for load balancing and ingress controller.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:143]: Fixed in 032f7000f4 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 20]
```diff
@@ -0,0 +1,278 @@
...
+
+**Service name prefix** column:
+
+You can fill in any valid value in this column.
+
+**Target** and **Port** column:
+
+The current offer has some restrictions on the T3 ports.
+
+* For a **Target** value of **admin-server-t3** you must use port 7005.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:161]: Fixed in 7bf081eed5 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 21]
```diff
@@ -0,0 +1,278 @@
...
+* For a **Target** value of **cluster-1-t3** you must use port 8011.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:162]: Fixed in 7bf081eed5 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 22]
```diff
@@ -0,0 +1,278 @@
...
+
+For the non-T3 ports, the recommended values are the usual 7001 for the **admin-server** and 8001 for the **cluster-1**.
+
+##### Application Gateway Ingress Controller
+
+This section enables you to create an Azure Application Gateway instance as the ingress controller of your WebLogic Server. This Application Gateway is pre-configured for end-to-end-SSL with TLS termination at the gateway using the provided SSL certificate and load balances across your cluster.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:168]: Fixed in db7f9ccb779 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 23]
```diff
@@ -0,0 +1,278 @@
...
+
+Select **Yes** or **No** for the option **Connect to Azure Application Gateway?** based on your preference. If you select **No**, you don't have to provide any details, and can proceed by selecting **Next : DNS Configuration >**. If you select **Yes**, you must specify the details required for the Application Gateway integration by entering the values for the fields as described next.
+
+You must select one of the following three options, each described in turn.
+
+* Upload a TLS/SSL certificate: Upload the pre-signed certificate now.
+* Identify an Azure Key Vault: The Key Vault must already contain the certificate and its password stored as secrets.
+* Generate a self-signed frontend certificate: generate a self-signed frontend certificate and apply it during deployment.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:176]: Fixed in 166e8763ac . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 24]
```diff
@@ -0,0 +1,278 @@
...
+
+**Upload a TLS/SSL certificate**
+
+| Field | Description |
+|-------|-------------|
+| Frontend TLS/SSL certificate(.pfx) | For information on how to create a certificate in PFX format, see [Overview of TLS termination and end to end TLS with Application Gateway](https://docs.microsoft.com/en-us/azure/application-gateway/ssl-overview). |
+| Password | The password for the certificate |
+| Confirm password | Re-enter the value of the preceding field. |
+| Trusted root certificate(.cer, .cert) | A trusted root certificate is required to allow backend instances in the application gateway. The root certificate is a Base-64 encoded X.509(.CER) format root certificate. |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:185]: Fixed in 166e8763ac . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 25]
```diff
@@ -0,0 +1,278 @@
...
+| Service Principal | A Base64 encoded JSON string of a service principal for the selected subscription. You can generate one with command `az ad sp create-for-rbac --sdk-auth | base64 -w0`. For more information see [Create a service principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli#create-a-service-principal). |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:186]: image

Is this ok?

File: documentation/3.3/content/userguide/aks/_index.md [hunk 26]
```diff
@@ -0,0 +1,278 @@
...
+| Name of the Azure KeyVault containing secrets for the Certificate for SSL Termination | Enter the name of the Azure Key Vault that stores the application gateway SSL certificate and the data required for SSL termination. |
+| The name of the secret in the specified KeyVault whose value is the SSL Certificate Data | Enter the name of the Azure Key Vault secret that holds the value of the SSL certificate data. |
+| The name of the secret in the specified KeyVault whose value is the password for the SSL Certificate | Enter the name of the Azure Key Vault secret that holds the value of the SSL certificate password. |
+| Service Principal | A Base64 encoded JSON string of a service principal for the selected subscription. You can generate one with command `az ad sp create-for-rbac --sdk-auth | base64 -w0`. For more information see [Create a service principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli#create-a-service-principal). |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:196]: image

File: documentation/3.3/content/userguide/aks/_index.md [hunk 27]
```diff
@@ -0,0 +1,278 @@
...
+| Field | Description |
+|-------|-------------|
+| Trusted root certificate(.cer, .cert) | A trusted root certificate is required to allow backend instances in the application gateway. The root certificate is a Base-64 encoded X.509(.CER) format root certificate. |
+| Service Principal | A Base64 encoded JSON string of a service principal for the selected subscription. You can generate one with command `az ad sp create-for-rbac --sdk-auth | base64 -w0`. For more information see [Create a service principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli#create-a-service-principal). |
+
+Regardless of how you provide the certificates, there are several other options when configuring the Application Gateway, as described next.
+
+| Field | Description |
+|-------|-------------|
+|Enable cookie based affinity | Select this box to enable cookie based affinity (sometimes called "sticky sessions". For more information, see [Enable Cookie based affinity with an Application Gateway](https://docs.microsoft.com/en-us/azure/application-gateway/ingress-controller-cookie-affinity). |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:209]: Fixed in 1943f09a0d . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 28]
```diff
@@ -0,0 +1,278 @@
...
+| Create ingress for Administration Console. | Select **Yes** to create an ingress for the admin console with the path `/console`. |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:210]: Fixed in 418a851d30 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 29]
```diff
@@ -0,0 +1,278 @@
...
+| Create ingress for WebLogic Remote Console. | Select **Yes** to create an ingress for the remote console with the path `/remoteconsole`. |
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:211]: Fixed in f3cb5e349b . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 30]
```diff
@@ -0,0 +1,278 @@
...
+
+When you are satisfied with your selections select **Next : DNS Configuration**.
+
+#### DNS Configuration
+
+The **DNS Configuration** blade enables you to provision the Oracle WebLogic Server Administration Console using a custom DNS name.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:217]: Fixed in db7f9ccb779 . Not yet pushed.

File: documentation/3.3/content/userguide/aks/_index.md [hunk 31]
```diff
@@ -0,0 +1,278 @@
...
+
+Select **Yes** or **No** for the option Configure Custom DNS Alias? based on your preference. If you select **No**, you don't have to provide any details, and can proceed by selecting **Next : Database >**. If you select **Yes**, you must choose either to configure a custom DNS alias based on an existing Azure DNS zone, or create an Azure DNS zone and a custom DNS alias. This can be done by selecting **Yes** or **No** for the option **Use an existing Azure DNS Zone**.
```
[the remark this answers is not in the document]
Comment (edburns) [reply documentation/3.3/content/userguide/aks/_index.md:219]: Fixed in 685f7d0809 . Not yet pushed.

**Comment by edburns** [comment]
Pushed these


~/workareas/weblogic-kubernetes-operator$ gitlogcom 40
commit 685f7d080958cb7c19b20d899f18f51b1f9bf628
commit f3cb5e349b4dc24ca088851a027c9dee27c15bab
commit 418a851d30065a0341b02d466d4b01adfd8a1673
commit 1943f09a0d74271e43d43f5efa74f30993107f59
commit 166e8763acc928c9b9a118b8b2c070d6b510c4e0
commit 7bf081eed57d96314fe4995db6cf7dbef01d30a3
commit 032f7000f4bb61a8f5564c47d43684ece57eed9d
commit 19e6749607cd46dcd007519f6de38a36986e0490
commit 362b24c75d871e84009c89192ebb0887f7953753
commit ea62f8619291d2f32e5ab46bfa8e49a5084f5f1c
commit 46f4fd0190b248874fd86c77b742a85aed45ab7d
commit 157d8f9724d3d9be758482732b63a45d277d4a81
commit db7f9ccb779ee8b96e78831811d7a68aaace5a82
commit 448ec221b9955fa1844ce02e27f40e49eb7f9c0b
commit a82ff8f4982d4077507b5a03ed2a52906c967d36
commit b26c5f98d1ce10ceef06d90a54722a19871cb372
commit c98ba6ac55f707bb5de539e33e16cac425f8faca
commit 7477e9be6267642aed92f13bda72c9dec35cd97f
commit e15fc2aec616637679aa43f88ded745863909373
commit 65707863d13c45ef7bcacc07b289de8d659bbbcb
commit 6ea4a9865bfad8b2dadf095bfd05478a1bf51d20
commit 8d914df0d61da575b1b5be1d5f3e92dc9975c9cd
commit 7b434c639402e1c6d03bd24c0f037e15c857df15
commit 211801fa71d70e4196b61865ecc759053261ada9

File: documentation/3.3/content/userguide/aks/_index.md (hunk 25 above)
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/3.3/content/userguide/aks/_index.md]: Yes.

**Comment by edburns** [comment]
@rosemarymarano @mriccell do we need anything else on this?

**Comment by rosemarymarano** [comment]
@edburns Not from my standpoint.

**Review by rosemarymarano — APPROVED** [review]

**Review by mriccell — APPROVED** [review]

**Pull request merged**